pkg/discovery/globalnet/globalnet.go

```diff
@@ -460,3 +462,41 @@ func ValidateExistingGlobalNetworks(kubeClient kubernetes.Interface, namespace s
 
 	return nil
 }
+
+func AllocateAndUpdateGlobalCIDRConfigMap(clusterID string, brokerAdminClientset *kubernetes.Clientset, brokerNamespace string,
+	netconfig *Config, status reporter.Interface) error {
+	status.Start("Discovering multi cluster details")
```

Comment:
Since we leave it up to the caller to end the status, I think we let the caller start it.